src/backend/storeManagers/storeManagerCommon/games.ts

```diff
@@ -160,7 +161,8 @@ export async function launchGame(
   const gameSettingsOverrides = await GameConfig.get(appName).getSettings()
   if (
     gameSettingsOverrides.targetExe !== undefined &&
-    gameSettingsOverrides.targetExe !== ''
+    gameSettingsOverrides.targetExe !== '' &&
+    fs.existsSync(gameSettingsOverrides.targetExe)
```

Comment:
Instead of using this one from `fs`, import `existSync` from `graceful-fs` like we are doing on other methods. The graceful package seems more reliable than the default from node.

Reply:
addressed in the latest commit